packages/sentinel-embedded/index.test.js

```diff
@@ -1,4 +1,5 @@
 import { render, waitFor } from '@testing-library/react'
+import { setImmediate } from 'timers'
```

Comment:
`setImmediate` is no longer polyfilled by jest, so importing the native one explicitly.